### PR TITLE
Enable ESLint caching

### PR DESCRIPTION
**Pull request by devgioele**

### CODE REVIEW

File: package.json
Comment (lukasvice) [inline package.json:23]: We should check why this is happening. When I format the document, it is split into several lines again. I don't know what is causing this, though, since Prettier should handle this always the same way.

Comment (lukasvice) [reply package.json:23]: I'll open another PR for this after you merged this one.

**Review by lukasvice — APPROVED** [review]

File: package.json
Comment (devgioele) [reply package.json:23]: Yeah idk. I am using prettier v2.7.1

**Comment by devgioele** [comment]
*Note: This PR was merged, but GitHub does not recognize it due to some bug after double-clicking the merge button.*

**Comment by lukasvice** [comment]
I was wondering about that, but saw that it was merged somehow :)

**Pull request closed without merging**
